src/clr-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -626,20 +616,23 @@
     flex: 0 0 auto;
     display: flex;
     flex-flow: row nowrap;
-    justify-content: flex-end;
```

Comment:
I think this change is unnecessary, the content is pushed full size by other properties.

Reply:
If the grid gets reeeeeally thin, this makes last attempt to keep it readable by keeping it centered and eating the outside margins instead of bleeding to the left.